bot/code_review_bot/__init__.py

```diff
@@ -132,13 +139,15 @@ def build_hash(self):
         # 1. lines affected by patch
         # 2. without any spaces around each line
         file_lines = file_content.splitlines()
-        start = self.line - 1  # file_lines start at 0, not 1
-        raw_content = "\n".join(
-            [
-                line.strip()
-                for line in file_lines[start : start + self.nb_lines]  # noqa E203
-            ]
-        )
+        if self.line is None or self.nb_lines is None:
+            # Use full file when line is not specified
+            lines = file_lines
+
+        else:
+            # Use a range of lines
+            start = self.line - 1  # file_lines start at 0, not 1
+            lines = file_lines[start : start + self.nb_lines]  # noqa E203
```

Comment:
Add E203 to .flake8 ignore (https://github.com/psf/black/issues/315#issuecomment-395457972), so we don't have to use `noqa`.